eng/runHelixAndNonHelixInParallelNonWindows.ps1

```diff
@@ -0,0 +1,38 @@
+    [CmdletBinding(PositionalBinding = $false)]
+    Param(
+        [string] $configuration,
+        [string] $buildSourcesDirectory,
+        [string] $customHelixTargetQueue,
+        [string] $officialBuildIdArgs,
+        [switch] $test
+    )
+
+    if (-not $test)
+    {
+        Write-Output "No '-test' switch. Skip both helix and non helix tests"
+        return
+    }
+
+    $runTestsCannotRunOnHelixArgs = ("-configuration", $configuration, "-ci", $officialBuildIdArgs)
+    $runTestsOnHelixArgs = ("-configuration", $configuration,
+     "-ci",
+    "-restore",
+    "-test",
+    "-projects", "$buildSourcesDirectory/src/Tests/UnitTests.proj",
+    "/bl:$buildSourcesDirectory/artifacts/log/$configuration/TestInHelix.binlog",
+    "/p:_CustomHelixTargetQueue=$customHelixTargetQueue")
+
+   $runTests = ("&'$PSScriptRoot/runTestsCannotRunOnHelix.sh' $runTestsCannotRunOnHelixArgs", "&'$PSScriptRoot/common/build.sh' $runTestsOnHelixArgs")
+
+   $runTests | ForEach-Object -Parallel { Invoke-Expression $_}
+
+   # An array of names of processes to stop on script exit
+   $processesToStopOnExit =  @('msbuild', 'dotnet', 'vbcscompiler')
+   Stop-Processes
+
+   function Stop-Processes() {
```

Comment:
Looking at the CI failure this might need to be declared above the invocation, or you could just not use it like a function at all.